c/test/legacy/auto_check_sbp_signing_MsgEd25519Signature.c

```diff
@@ -691,7 +692,8 @@ START_TEST(test_legacy_auto_check_sbp_signing_MsgEd25519Signature) {
       test_msg_len += sizeof(test_msg->signed_messages[0]);
     }
     test_msg->signed_messages[24] = 10616;
-    sbp_payload_send(&sbp_state, 0xC01, 9876, test_msg_len, test_msg_storage,
+    test_msg->stream_counter = 1;
+    sbp_payload_send(&sbp_state, 0xC03, 66, test_msg_len, test_msg_storage,
                      &dummy_write);
```

Comment:
I undestand that the change from 0xC01 to 0xC03 comes from the new msg type definition, can you explain where the change from 9876 to 66 comes from?

Reply:
When I regenerated the test yaml with json2test it used 42 as the sender id https://github.com/swift-nav/libsbp/blob/c0327571b0bc3fb479f4bfaf25911b3d8a45a87d/spec/tests/yaml/swiftnav/sbp/signing/test_MsgEd25519Signature.yaml#L143